makefiles/defaultmodules.inc.mk

```diff
@@ -1,6 +1,7 @@
 DEFAULT_MODULE += board cpu core core_init core_msg core_panic sys
 
 DEFAULT_MODULE += auto_init
+DEFAULT_MODULE += auto_init_actuators_sensors_default
```

Comment:
Does it make sense to have this as the default?
Existing applications don't expect any auto initializing sensors.

Reply:
I dropped the commit making it a default module